### ISSUE TITLE
Inconsistent API responses for create and update operations

### ISSUE THREAD

**Issue by phpai** [openkfw/TruBudget#103]
In GitLab by @kevinbader on Mar 6, 2019, 08:39

## Description

Some responses look like this:

```json
{
  "apiVersion": "1.0",
  "data": {
    "created": "true"
  }
}
```

Others look like this (note that here `data` is not an object):

```json
{
  "apiVersion": "1.0",
  "data": "OK"
}
```

Neither are useful and together they make for an inconsistent API.

## Proposal

Either use `204 No content` or return a reference to the updated entity in the body.

**Comment by Stezido** [comment]
Returning "data" as empty object with response code 200 is the way to go here because we do not create or update resources like RESTful services so we should not use 201/204 codes . Future changes will also be easier to handle.

**Comment by Stezido** [comment]
solved by #154
